<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patience_sa</class>
 <widget class="QWidget" name="patience_sa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#patience_sa{background-image: url(:/hospital.jpg);}</string>
  </property>
  <widget class="QPushButton" name="b">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="office">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="phonenumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="borndate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="gender">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>医生编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="blood_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="docnumber_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modify">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>550</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="borntime_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>病历号</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="age">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>血型</string>
   </property>
  </widget>
  <widget class="QLabel" name="history">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>既往病史</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="history_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>230</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="end">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>诊断结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="number_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>看诊时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>160</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
